tox.ini

```diff
@@ -3,9 +3,9 @@ envlist = py35, py36, py37, black, docs
 
 [travis]
 python =
-  3.5: py35
   3.6: py36
-  3.7: py37, docs, black, flake8
+  3.7: py37
```

Comment:
baffled by how this was building before because the py36 envt didn't exist in tox i think... i may need to learn more about how tox works but for now this seems to work